### PR TITLE
Fixed wrong Predis Mock

### PR DESCRIPTION
**Pull request by gianarb**
ping @Ocramius

### CODE REVIEW

File: tests/DoctrineModuleTest/Service/CacheFactoryTest.php [hunk 1]
```diff
@@ -76,9 +76,8 @@ public function testCreatePredisCache()
             ]
         )->setService(
             'my_predis_alias',
-            $this->getMock('Predis\Client')
+            $this->getMock('Predis\ClientInterface')
```
Comment (Ocramius) [inline tests/DoctrineModuleTest/Service/CacheFactoryTest.php]: Can only work with doctrine/cache 1.5.0: change the composer.json accordingly, please.

Comment (gianarb) [inline tests/DoctrineModuleTest/Service/CacheFactoryTest.php]: Can you help me to understand who require doctrine/cache? :) At the moment this module does not require it explicitly

Comment (gianarb) [inline tests/DoctrineModuleTest/Service/CacheFactoryTest.php]: Can I add doctrine\cache and stop?

Comment (Ocramius) [inline tests/DoctrineModuleTest/Service/CacheFactoryTest.php]: Yep

**Comment by gianarb** [comment]
@Ocramius can we go? :)

**Comment by Ocramius** [comment]
:+1:

**Pull request merged**